packages/main/src/Select.js

```diff
@@ -127,6 +127,7 @@ const metadata = {
  * @extends sap.ui.webcomponents.base.WebComponent
  * @tagname ui5-input
  * @public
+ * @since 0.8.0
```

Comment:
we could write it as "0.8.0 (next release)" here so it also is seen on  the properties? what do you think?

Reply:
I tried it in the beginning, the plugin can`t parse it - it stops at interval/space